app/views/finders/_option_select_facet.html.erb

```diff
@@ -1,12 +1,11 @@
 <% cache_if option_select_facet.cacheable?, option_select_facet.cache_key do %>
-  <% close_facet = option_select_facet.unselected? && ((option_select_facet_counter > 0) || option_select_facet.close_facet?) %>
   <%= render partial: 'components/option-select', locals: {
     :key => option_select_facet.key,
     :title => option_select_facet.name,
     :aria_controls_id => "js-search-results-info",
     :options_container_id => option_select_facet.key,
     :options => option_select_facet.options("js-search-results-info", option_select_facet.key),
-    :closed_on_load => close_facet,
+    :closed_on_load => option_select_facet_counter > 0 && option_select_facet.unselected?,
```

Comment:
@ChrisBAshton How does this work in AAIB reports? Does changing this conditional affect other finders?

Reply:
This conditional is the same as before (https://github.com/alphagov/finder-frontend/pull/1073/files#diff-4d0b8cfd231c9d5050ca9adf67e518d4L2), except there is no call to `close_facet?`, so no check for the number of options.

But this is ok because the 10 option limit is a piece of legacy code that can be removed - see the commit message in 77a0b9370102e44e755bbeed3510f0c410e7783e.